docs/topics/rn-new-features.adoc

```diff
@@ -8,35 +8,31 @@
 
 This section describes the new features of the {ProductName} ({ProductShortName}) {ProductVersion}.
 
-[IMPORTANT]
-====
-Rules for migrating from Java EE8 to Jakarta EE9 and OpenRewrite support are both provided as Technology Preview only. Technology Preview features are not supported with Red Hat production service level agreements (SLAs), might not be functionally complete, and Red Hat does not recommend to use them for production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
-
-See link:{KBArticleTechnologyPreview}[Technology Preview features support scope] on the Red&nbsp;Hat Customer Portal for information about the support scope for Technology Preview features.
-====
-
-.Rules for migrating from Java EE8 to Jakarta EE9
-{ProductShortName} rules replace Java EE8 artifact dependencies and package import statements and rename XML schema namespaces, prefixed properties, and bootstrapping files for Jakarta EE9. You run these rules by specifying the `jakarta-ee` target.
+.Analyze Known Libraries
+You can now use the `analyzeKnownLibraries` advanced option to analyze libraries embedded within your applications. By default, MTA's default behavior is to only analyze application code, rather than libraries that are declared as dependencies. This option extends your ability to analyze your applications.
 
```

Comment:
@RichardHoch 
Why do you need the second 'default'? Also, use {ProductShortName] instead of MTA - here and throughout. "By default, {ProductShortName] 's behavior is ..."

Reply:
Fixed.